tests/generic_config_updater/test_incremental_qos.py

```diff
@@ -241,11 +241,15 @@ def test_incremental_qos_config_updates(duthost, tbinfo, ensure_dut_readiness, c
         output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
         if op == "replace" and not field_value:
             expect_op_failure(output)
-
-        if is_valid_platform_and_version(duthost, "BUFFER_POOL", "Shared/headroom pool size changes", op, field_value):
-            expect_op_success(duthost, output)
-            ensure_application_of_updated_config(duthost, configdb_field, value)
         else:
-            expect_op_failure(output)
+            if is_valid_platform_and_version(duthost,
```

Comment:
minor comment: use `elif`

Reply:
For non replace op, it requires validation, which is not a parallel check with op. So, for readability, I would prefer keep in current way.